src/demo/example-data/deckgl-map.json

```diff
@@ -62,6 +62,11 @@
                     "valueRange": 731.0,
                     "opacity": 1.0,
                     "image": "https://i.imgur.com/Dx8eseE.png"
+                },
+                {
+                    "@@type": "WellsLayer",
+                    "id": "wells-layer",
+                    "data": "src/demo/example-data/emerald_wells.json"
```

Comment:
These wells are not overlapping with the surface, so maybe you can get some example wells from the same project as the surface above (the surface will be properly placed after #308 is merged). I think @Midtveit posted the RMS project somewhere.

Reply:
@alexandruandrei91 replaced Emerald wells with wells from Volve project in the latest commit.